src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -22,6 +22,9 @@
  */
 class Filesystem
 {
+    const FILE_TYPE_REGULAR = 'regular';
```

Comment:
should be public

Reply:
I wanted to keep it compatible with PHP 5.6 (default being public), but as the composer.json forces ^7.1, it makes sense. I added the visibility.

Reply:
Thank you

Reply:
Thank you